pkg/mesh/app.go

```diff
@@ -125,27 +60,6 @@ func (kr *KRun) WaitTCPReady(addr string, max time.Duration) error {
 	return nil
 }
 
-// WaitAppStartup waits for app to be ready to accept requests.
-// - default is KNative 'listen on the app port' ( 8080 default, PORT_http overrides )
-// - startupProbe.tcp and startupProbe.http can define alternate port and using http ready.
-func (kr *KRun) WaitAppStartup() error {
```

Comment:
This is also intended to deal with the multi-container - i.e. we still want to wait for the app ( maybe in a different container ) listening to a port before declaring envoy ready.